lib/vmdb/gettext/debug.rb

```diff
@@ -0,0 +1,31 @@
+require 'fast_gettext'
+
+# include this module to see translations in the UI
+module Vmdb
+  module Gettext
+    module Debug
+      DL = "\u00BB".encode("UTF-8")
+      DR = "\u00AB".encode("UTF-8")
+
+      # slightly modified copy of fast_gettext _ method
+      def _(key)
+        DL + FastGettext._(key) + DR
```

Comment:
Prefer interpolation over `+`.  Even though it's for debug, this will result in 2-3x as many Strings being allocated.

```ruby
"#{DL}#{FastGettext._(key)}#{DR}"
```